src/NServiceBus.RavenDB/Gateway/RavenDbGatewayDeduplicationSettingsExtensions.cs

```diff
@@ -27,6 +31,10 @@ public static PersistenceExtensions<RavenDBPersistence> UseDocumentStoreForGatew
         /// </summary>
         /// <param name="cfg"></param>
         /// <param name="storeCreator">A Func that will create the document store on NServiceBus initialization.</param>
+        [ObsoleteEx(
+            Message = "RavenDB gateway persistence has been moved to the NServiceBus.Gateway.RavenDB dedicated package.",
+            RemoveInVersion = "9.0.0",
+            TreatAsErrorFromVersion = "7.0.0")]
```

Comment:
@DavidBoike is this correct or should it be?

- TreatAsErrorFromVersion = "7.0.0"
- RemoveInVersion = "8.0.0"

Reply:
Currently on Major 6, so as you have it here is correct:

* Obsoletes with warn in 6
* Obsoletes with error in 7
* Removed in 9

Reply:
I can't count. 8 does exist. Should be RemoveInVersion = 8.